src/components/MobileSideMenu/scss/Styles.scss

```diff
@@ -0,0 +1,46 @@
+$mobile-side-menu-content-width: 70%;
+
+$overlay-background-color: transparentize(#4e4949, 0.2);
+
+.mobile-side-menu {
+
+}
+
+.mobile-side-menu__overlay {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  bottom: -100%;
+  background-color: $overlay-background-color;
+  left: 0;
+  z-index: 99999;
+  animation: menu-overlay-slide 0.25s forwards;
+}
+
+@keyframes menu-overlay-slide {
+  100% { bottom: 0; }
+}
+
+.mobile-side-menu__content {
+  position: fixed;
+  z-index: 999999;
+  left: calc(-1 * #{$mobile-side-menu-content-width}) ;
+  width: $mobile-side-menu-content-width;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  animation: menu-content-slide 0.25s forwards;
+  background-color: white;
+}
+
+@keyframes menu-content-slide {
+  100% { left: 0; }
```

Comment:
it's not a great idea to use `left/right/top/bottom` in animations-
rather use `transform` here-

see https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/

Reply:
done